src/Container/InputPage/TextArea/TextArea.module.css

```
@@ -0,0 +1,15 @@
+.textarea{
+    padding: 0.5rem;
+    outline: 0px;
+    border:0px;
+    width: 25rem;
+    height: 10rem;
+    resize: none;
+    letter-spacing: 1px;
+    background-color: var(--neutral-mild);
+    outline: 0px;
```

```suggestion
.textarea{
    padding: 0.5rem;
    outline: 0;
    border:0;
    width: 25rem;
    height: 10rem;
    resize: none;
    letter-spacing: 1px;
    background-color: var(--neutral-mild);
```
Hi Muthu, I am suggesting a simple refactoring here. Removed the units follwowing 0.